### PR TITLE
Changed usage of image_loader to image_id_extractor

### PR DESCRIPTION
**Pull request by ArthurRab**
Systems with multiple instances of the same image name/tag were having
issues. New script uses the id instead.

Also updated rules_docker commit in use and dependencies.
Removed image_loader script and its tests.

### CODE REVIEW

File: package_managers/install_pkgs.bzl [hunk 1]
```diff
@@ -162,8 +163,8 @@ _attrs = {
         allow_files = True,
         single_file = True,
     ),
-    "_image_loader": attr.label(
-      default = "//util:image_loader.sh",
+    "_image_id_extractor": attr.label(
+        default = "@io_bazel_rules_docker//contrib:extract_image_id.sh",
```
Comment (xingao267) [inline package_managers/install_pkgs.bzl]: extra space

**Comment by xingao267** [comment]
@tejal29  could you take a look at this. We made this change because we are seeing tag naming conflict in our CI when multiple instances of download_pkgs/install_pkgs run at the same time. All the images pulled through container_pull rule will have tag bazel/image:image. So we switch to use the image id instead.

**Comment by xingao267** [comment]
@tejal29 friendly ping on this one~

**Comment by xingao267** [comment]
@tejal29 Thanks for approving! Please merge whenever you think is ready.

**Comment by tejal29** [comment]
@xingao267 feel free to ping me on Gchat.

**Pull request merged**